pythonforandroid/toolchain.py

```diff
@@ -376,6 +376,16 @@ def __init__(self):
             dest='local_recipes', default='./p4a-recipes',
             help='Directory to look for local recipes')
 
+        generic_parser.add_argument(
+            '--activity-package', '--activity_package',
+            dest='activity_package', default='org.kivy.android',
+            help='The java package where activity class is defined')
+
+        generic_parser.add_argument(
+            '--activity-class-name', '--activity_class_name',
+            dest='activity_class_name', default='PythonActivity',
```

Comment:
I'm wondering if both default couldn't be shared with the one defined in `build.py` to keep a single source of truth. Maybe using a const somewhere.
Also matter of taste but I would ditch the underscored version of the argument

Reply:
i will check about unit tests Andre.  i was a bit in the rush when started it.

Reply:
Thanks, we have pretty low coverage in both p4a and buildozer, but I'm trying to push for increasing it slowly

Reply:
added some basic assertion in the existing unit test to check that Context was populated properly with new value of `activity_class_name`